examples/pytorch/mixhop/README.md

```diff
@@ -0,0 +1,99 @@
+# DGL Implementation of MixHop
+
+This DGL example implements the GNN model proposed in the paper [MixHop: Higher-Order Graph Convolution Architectures via Sparsified Neighborhood Mixing](https://arxiv.org/abs/1905.00067). For the original implementation, see [here](https://github.com/samihaija/mixhop).
+
+## Example implementor
+
+This example was implemented by [xnuohz](https://github.com/xnuohz) during his intern at the AWS Shanghai AI Lab.
```

Comment:
I think you can simply replace this section with something like below

```
Contributor: [Name](github link)
```

Reply:
Done.